File: BackTracking/Easy5_printBitwise.py
def PrintBitwise(n, arr = None):
    if(arr == None):
        arr = [None] * n
    
    if(n <1):
        print(arr)
        return
    else:
        arr[n-1] = 0
        PrintBitwise(n-1, arr)
        arr[n-1] = 1
        PrintBitwise(n-1, arr)


PrintBitwise(3)

File: BackTracking/Easy5_printBitwise.js
function PrintBitwise(n, array = new Array(n)){

    if(n < 1){
        console.log(array);
        return;
    }

    array[n-1] = 0;
    PrintBitwise(n-1, array);
    array[n-1] = 1;
    PrintBitwise(n-1, array);
}


PrintBitwise(2)
